angular.json

```diff
@@ -93,17 +100,6 @@
               "src/favicon.ico"
             ]
           }
-        },
-        "lint": {
```

Comment:
is there some other way to run the linter now? Why the previous one had to be deleted?